pkg/oci/utils.go

```diff
@@ -39,6 +39,16 @@ var (
 	BundlePathKey = "com.github.containers.virtcontainers.pkg.oci.bundle_path"
 )
 
+type CompatOCIProcess struct {
```

Comment:
Agreed. In particular, a mention to opencontainers/runtime-spec@37391fb in a comment would be helpful.
@sboeuf, can you quickly do that to make things a little clearer ?